cloudbuild.prod.yaml

```diff
@@ -0,0 +1,56 @@
+# See list of built-in substitutions at
+# https://cloud.google.com/container-builder/docs/configuring-builds/substitute-variable-values
+substitutions:
+  _CLOUDSDK_COMPUTE_REGION: us-central1
+  _CLOUDSDK_CONTAINER_CLUSTER: prod-gke
+  _ENVIRONMENT: prod
+  _GCLOUD_PROJECT: etcd-io
+  _HELM_CHART_GIT_REPO: https://github.com/etcd-io/discovery.etcd.io.git
+  _HELM_CHART_GIT_BRANCH: master
+  _HELM_CHART_PATH: kubernetes/helm
+  _K8S_NAMESPACE: discoveryserver
+  _REPO_NAME: discoveryserver
+
+steps:
+- name: 'gcr.io/cloud-builders/git'  # Clone infrastructure repo
+  args: ['clone', '${_HELM_CHART_GIT_REPO}', '/helm']
+  volumes:
+  - name: 'helm'
+    path: '/helm'
+- name: 'gcr.io/cloud-builders/git'
+  args: ['checkout', '${_HELM_CHART_GIT_BRANCH}']
+  dir: '/helm'
+  volumes:
+    - name: 'helm'
+      path: '/helm'
+- name: 'gcr.io/cloudkite-public/gcloud-helm3:20191117'  # Deploy helm chart
+  args:
+  - /usr/local/bin/helm
+  - upgrade
+  - --wait
```

Comment:
Should we add `--atomic` ?

Reply:
Yes, let's add it. There are potential benefits having it.